### PR TITLE
Release 8.6.0

### PR DESCRIPTION
**Pull request by tannalynn**
_Before contributing, please read our [contributing guidelines](https://github.com/newrelic/newrelic-ruby-agent/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/newrelic/.github/blob/master/CODE_OF_CONDUCT.md)._

# Overview
Describe the changes present in the pull request

# Related Github Issue
Include a link to the related GitHub issue, if applicable

# Testing
The agent includes a suite of unit and functional tests which should be used to
verify your changes don't break existing functionality. These tests will run with 
Github Actions when a pull request is made. More details on running the tests locally can be found 
[here for our unit tests](https://github.com/newrelic/newrelic-ruby-agent/blob/main/test/README.md), 
and [here for our functional tests](https://github.com/newrelic/newrelic-ruby-agent/blob/main/test/multiverse/README.md).
For most contributions it is strongly recommended to add additional tests which
exercise your changes. 

# Reviewer Checklist
- [ ] Perform code review
- [ ] Add performance label
- [ ] Perform appropriate level of performance testing
- [ ] Confirm all checks passed
- [ ] Add version label prior to acceptance

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # New Relic Ruby Agent Release Notes #
 
+  ## v8.6.0
+
+  * **Telemetry-in-Context: a quick way to view logs no matter where you are in the platform**
```
Comment (kaylareopelle) [inline CHANGELOG.md:5]: I think we have to change the title slightly, even though the google doc draft shows something different. this was decided in a Slack thread: 

```suggestion
* **Telemetry-in-Context: Automatic Application Logs, a quick way to view logs no matter where you are in the platform**
```

**Review by kaylareopelle — DISMISSED** [review]
Found one update for the CHANGELOG. Everything else looks good!

File: CHANGELOG.md (hunk 1 above)
Comment (tannalynn) [reply CHANGELOG.md:5]: Thanks for letting me know! I was going off the google doc, so I didn't see this

**Review by kaylareopelle — APPROVED** [review]
Thanks for the update!

**Pull request merged**